charts/seed-controlplane/charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -132,19 +146,31 @@ spec:
         volumeMounts:
         - mountPath: /var/etcd/data
           name: etcd-{{ .Values.role }}
+        - name: ca
+          mountPath: /srv/kubernetes/ca
+        - mountPath: /var/etcd/ssl
```

Comment:
Can we reverse the order (first `name` and then `mountPath`) to improve readability?